packages/compass-aggregations/src/components/pipeline/pipeline.jsx

```diff
@@ -239,6 +240,14 @@ class Pipeline extends PureComponent {
     return <PipelineToolbar />;
   }
 
+  renderPipelineWorkspace() {
+    return this.props.workspace === 'results' ? (
+      <PipelineResultsWorkspace />
+    ) : (
+      <PipelineBuilderWorkspace {...this.props} />
```

Comment:
Are all these props used by `PipelineBuilderWorkspace` component?

Reply:
definitely no. wanted to keep the existing implementation as is. but now as I'm digging into these props, I may've to   fix this.